### PR TITLE
feat: add short flags for branch and git

### PR DESCRIPTION
**Pull request by SirWindfield**

### CODE REVIEW

**Comment by SirWindfield** [comment]
This is probably a merge conflict with #233

File: src/lib.rs [hunk 1]
```diff
@@ -52,10 +52,10 @@ pub enum Cli {
 
 #[derive(Debug, StructOpt)]
 pub struct Args {
-    #[structopt(long = "git")]
```
Comment (k0pernicus) [inline src/lib.rs:55]: Is it possible to force 'g' here please?

File: src/lib.rs [hunk 2]
```diff
@@ -52,10 +52,10 @@ pub enum Cli {
...
+    #[structopt(short, long = "git")]
     git: String,
     // Branch to use when installing from git
-    #[structopt(long = "branch")]
```
Comment (k0pernicus) [inline src/lib.rs:58]: Is it possible to force 'b' here please?

**Comment by k0pernicus** [comment]
Ok, nevermind - this PR will be in conflict with #233 indeed

**Comment by k0pernicus** [comment]
Could be great if you can merge those changes in #233 :)

Thanks in advance!

**Comment by SirWindfield** [comment]
Superseeded by #233 .

**Pull request closed without merging**